deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -436,27 +416,30 @@ JsValueRef Utils::HasPropertyHandler(
   }
 
   if (isPropIntType) {
-    if (objectData->indexedPropertyQuery != nullptr) {
-      HandleScope scope(nullptr);
-      PropertyCallbackInfo<Integer> info(
-        *objectData->indexedPropertyInterceptorData,
-        reinterpret_cast<Object*>(object),
-        /*holder*/reinterpret_cast<Object*>(object));
-      objectData->indexedPropertyQuery(index, info);
-      if (info.GetReturnValue().Get() != JS_INVALID_REFERENCE) {
-        return jsrt::GetTrue();  // intercepted
+    if (objectData->setterGetterInterceptor != nullptr)
+    {
+      if (objectData->setterGetterInterceptor->indexedPropertyQuery != nullptr) {
```

Comment:
nit: lots of these if blocks have inconsistent curly-brace styles

Reply:
done